packages/tools/client-debugger/client-debugger-view/src/components/TelemetryView.tsx

```diff
@@ -104,10 +117,121 @@ export function TelemetryView(): React.ReactElement {
 		};
 	}, [messageRelay, setTelemetryEvents]);
 
+	/**
+	 * Interface for each item in the telemetry table.
+	 */
+	interface Item {
+		category: string;
+		eventName: string;
+		information: string;
+	}
+
+	const items: Item[] =
+		telemetryEvents !== undefined
+			? telemetryEvents?.map((message) => {
+					return {
+						category: message.logContent.category,
+						eventName: message.logContent.eventName,
+						information: JSON.stringify(
+							message.logContent,
+							jsonSerializationTransformer,
+							"  ",
```

Comment:
What is this for?

Reply:
@jikim-msft  It was a json serializer that removed values that were already being displayed ( logcontent.category/ eventName) before showing the json. But this is being deprecated and all json values in information will be shown.